classes/fields/pick.php

```diff
@@ -168,6 +168,28 @@ public function options() {
 				),
 				'dependency' => true,
 			),
+			static::$type . '_display_format_multi'   => array(
+				'label'      => __( 'Display Format', 'pods' ),
+				'help'       => __( 'Used as format for front-end display', 'pods' ),
+				'depends-on' => array( static::$type . '_format_type' => 'multi' ),
+				'default'    => 'default',
+				'type'       => 'pick',
+				'data'       => array(
+					'default' => __( 'Item, Item and Item', 'pods' ),
+					'custom'  => __( 'Custom separator', 'pods' ),
+				),
+				'dependency' => true,
+			),
+			static::$type . '_display_format_separator'   => array(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'_display_format_separator'" and double arrow, but found 3.

<sub>hash: edd772b7387f9fe9f075cfdbd8ba0f62</sub>